_data/capability-matrix.yml

```diff
@@ -0,0 +1,561 @@
+columns:
```

Comment:
In the future when the number of runners grows, it'd be good to break these out into separate files per runner. Ok for now as is.